disperser/controller/dispatcher.go

```diff
@@ -323,12 +332,15 @@ func (d *Dispatcher) HandleSignatures(
 	for _, key := range batchData.BlobKeys {
 		err := d.blobMetadataStore.UpdateBlobStatus(ctx, key, v2.GatheringSignatures)
 		if err != nil {
-			d.logger.Error("failed to update blob status for blob %s to gathering signatures",
+			d.logger.Error("failed to update blob status to 'gathering signatures'",
 				"blobKey", key.Hex(),
+				"batchHeaderHash", batchHeaderHash,
 				"err", err)
 		}
 	}
 
+	// submit an empty attestation before starting to gather signatures.
+	// a new attestation will be periodically resubmitted as signatures are gathered.
```

Comment:
curious why do you use the term "(re)submit" when talking about writing to dynamoDB? why not just say "updated" or "overwritten"? Is there some daemon process that manages the db that we "submit" write requests to?

Reply:
> Is there some daemon process that manages the db that we "submit" write requests to?

Nope, just a poor choice of term on my part. Fixed in afa7304f62d00a2e4689b56ca23e471237c27725